### ISSUE TITLE
Maybe ood-testdev in LXMERT is more flexible

### ISSUE THREAD

**Issue by alice-cool** [gqa-ood/GQA-OOD#3]
Dear scholar,
Because we always finetune the model in downstream tasks. But you select the validation ood split to define alpha. It is not flexible because it needs pretraining again by excluding the qa pairs from validation sets. Just suggestions.

**Comment by gqa-ood** [comment]
Dear @alice-cool,

Yes, I agree with you that it is not flexible to have to do the pretraining again.
But, we choose to validate on the validation and test on the test-dev because the GQA test-std is not public, so we cannot create a gqa-ood test-std set. 

At the same time, I do not really understand why qa pairs from the validation set are used for training!

Corentin K.

**Comment by alice-cool** [comment]
Dear scholar,
It is just the default setting for lxmert with high performance. Maybe you think 12587 samples from testdev is not good to make gqa-ood testdev set.
![image](https://user-images.githubusercontent.com/16870890/144528026-7df6b2af-7933-4f26-8bec-6f9eb51b9be2.png)


If you didn't train the validation set on training phase. so the LXMERT default hypermeter will be changed with validation accuracy, or wasn't changed just excluding validation set for training. Because I didn't find any hypermeter for LXMERT in gqa-ood testdev accuracy 49.8 at tail.

![image](https://user-images.githubusercontent.com/16870890/144528055-145fb8fa-dda0-4925-b194-d0389c3919a7.png)
